README.md

```diff
@@ -18,19 +19,28 @@ Kartograf: A 3D Atom Graph Mapper
 [![Pip Install](https://img.shields.io/badge/pip%20install-kartograf-d9c4b1)](https://pypi.org/project/kartograf/)
 [![Conda Install](https://img.shields.io/badge/Conda%20install---c%20conda--forge%20kartograf-009384)](https://anaconda.org/conda-forge/kartograf)
 
+Kartograf is a packgage for atom mappings focussing on 3D geometries.
+This package can be for example be used to generate hybrid topology systems, where an atom mapping is required to determine the core region of the approach.
+But of course there exist also other use cases for this package.
+The atom mapper takes two set of coordinates of molecules as input.
+Optionally those set of coordinates can be aligned onto each other, checkout the `atom_aligner` module functions 
+of Kartograf that offer a shape alignment implementation and a MCS-Sceleton alignment.
+The `atom_mapper` can be used to generate the 3D geometry focused atom mapping, the algorithm is described in the related publication of Kartograf (see reference).
+Additionally, rule based filter functions can be provided to demap atoms, that do not fullfill the desired criteria, see `filters`.
+Several mapping scoring metrics are provided, that evaluate geometric properties of your mapping, from `atom_mapping_scorer`, which might be useful for checking quality of your mappings.
+Finally, there is a visualization function `display_mappings_3d` that can be used to check out the mappings with a jupyter notebook widget.
 
-Kartograf offers a geometric atom mapper approach, that allows to map a given set of ligand coordinates. (can be used for hybrid topology  RBFE calculations)
-This package can be used standalone, or from the OpenFE environment.
+You can find our Preprint on [ChemRxiv](https://doi.org/10.26434/chemrxiv-2023-0n1pq) 
 
-**More will be here soon!**
 
 ## Usage
 ```python3
 from rdkit import Chem
-from kartograf.atom_align import align_mol_shape
+from kartograf.atom_aligner import align_mol_shape
+from kartograf.atom_mapping_scorer import MappingRMSDScorer
```

Comment:
I'd probably move everything you might expect people to import to the top level namespace so people don't have to remember that `align_mol_shape` lives inside `atom_aligner`.